src/pages/Home/Home.js

```diff
@@ -0,0 +1,15 @@
+import AboutSection from "./AboutSection";
+import MoodSection from "./MoodSection";
+import QouteSection from "./QuoteSection";
+import AuthorSection from "./AuthorSection";
```

Comment:
Since these components seemed to only relate to the `Home` page, I figured I'd move it into a folder with close proximity to the parent caller. I would only move this back up to the `src/components` directory if any of these files get imported into any of your other pages.